torchao/quantization/quant_primitives.py

```diff
@@ -41,6 +43,245 @@
     # TODO: need to clean up above functions
 ] + (_AFTER_TORCH_2_3_ONLY if TORCH_VERSION_AFTER_2_3 else [])
 
+
+_DTYPE_TO_QVALUE_BOUNDS = {
+    torch.uint8: (0, 255),
+    torch.int8: (-128, 127),
+    torch.int16: (-(2**15), 2**15 - 1),
+    torch.int32: (-(2**31), 2**31 - 1),
+}
+
+if TORCH_VERSION_AFTER_2_3:
+    _DTYPE_TO_QVALUE_BOUNDS.update({
+        torch.uint1: (0, 2**1-1),
+        torch.uint2: (0, 2**2-1),
+        torch.uint3: (0, 2**3-1),
+        torch.uint4: (0, 2**4-1),
+        torch.uint5: (0, 2**5-1),
+        torch.uint6: (0, 2**6-1),
+        torch.uint7: (0, 2**7-1),
+    })
+
+def _get_qmin_qmax(dtype, quant_min, quant_max):
```

Comment:
I feel this is more like `_check_qmin_qmax`? Alternatively make `quant_min` and `quant_max` default to None?

Reply:
yeah this actually combined two functions, i can split them as well

Reply:
we are thinking about just don't allow quant_min/quant_max, will update this after we made a decision here, I'll add a TODO here